concepticondata/conceptlists/Steinthal-1875-1565.tsv

```diff
@@ -0,0 +1,1566 @@
+Steinthal-1875-1565-313	313	kleines Mädchen	kleines Mädchen	little girl	1646	GIRL
+Steinthal-1875-1565-314	314	reife Jungfrau	reife Jungfrau	young woman	77	YOUNG WOMAN
+Steinthal-1875-1565-315	315	Gatte	Gatte	husband	1200	HUSBAND
+Steinthal-1875-1565-316	316	Gattin	Gattin	wife	1199	WIFE
+Steinthal-1875-1565-317	317	Braut	Braut	bride	1266	BRIDE
+Steinthal-1875-1565-318	318	Bräutigam	Bräutigam	bridegroom	3282	GROOM
+Steinthal-1875-1565-319	319	Junggeselle	Junggeselle	bachelor		
+Steinthal-1875-1565-320	320	Wittwe	Wittwe	widow	1756	WIDOW
+Steinthal-1875-1565-321	321	Wittwer	Wittwer	widower	1981	WIDOWER
+Steinthal-1875-1565-322	322	sich verheirathen	sich verheirathen	to marry	203	MARRY
+Steinthal-1875-1565-323	323	Ein Mann A hat die Jungfrau B geheirathet.	Ein Mann A hat die Jungfrau B geheirathet.	A man A has married the virgin B.		
+Steinthal-1875-1565-324	324	Sein Sohn hat meine Tochter zur Frau.	Sein Sohn hat meine Tochter zur Frau.	His son is married to my daughter.		
+Steinthal-1875-1565-325	325	Heirath	Heirath	marriage	2432	MARRIAGE
+Steinthal-1875-1565-326	326	Hochzeit	Hochzeit	wedding	1259	WEDDING
+Steinthal-1875-1565-327	327	Er ist unverheirathet.	Er ist unverheirathet.	He is unmarried.		
+Steinthal-1875-1565-328	328	Vater	Vater	father	1217	FATHER
+Steinthal-1875-1565-329	329	Mutter	Mutter	mother	1216	MOTHER
+Steinthal-1875-1565-330	330	schwanger	schwanger	pregnant	1123	PREGNANT
+Steinthal-1875-1565-331	331	saugen	saugen	to suckle		
+Steinthal-1875-1565-332	332	säugen	säugen	to breastfeed	2472	SUCKLE (BREASTFEED, NURSE)
+Steinthal-1875-1565-333	333	Säugling	Säugling	infant	1289	BABY
+Steinthal-1875-1565-334	334	Bruder (älterer, jüngerer)	Bruder	brother	1262	BROTHER
+Steinthal-1875-1565-335	335	Bruder (älterer, jüngerer)	älterer Bruder	older brother	1759	OLDER BROTHER
+Steinthal-1875-1565-336	336	Bruder (älterer, jüngerer)	jüngerer Bruder	younger brother	1760	YOUNGER BROTHER
+Steinthal-1875-1565-337	337	Schwester (ältere, jüngere)	Schwester	sister	1263	SISTER
+Steinthal-1875-1565-338	338	Schwester (ältere, jüngere)	ältere Schwester	older sister	1758	OLDER SISTER
+Steinthal-1875-1565-339	339	Schwester (ältere, jüngere)	jüngere Schwester	younger sister	1761	YOUNGER SISTER
+Steinthal-1875-1565-340	340	Grossvater (von Vaters, von Mutters Seiten)	Großvater	grandfather	1383	GRANDFATHER
+Steinthal-1875-1565-341	341	Grossvater (von Vaters, von Mutters Seiten)	Großvater väterlicherseits	paternal grandfather	2560	PATERNAL GRANDFATHER
+Steinthal-1875-1565-342	342	Grossvater (von Vaters, von Mutters Seiten)	Großvater mütterlicherseits	maternal grandfather	2178	MATERNAL GRANDFATHER
+Steinthal-1875-1565-343	343	Grossmutter	Großmutter	grandmother	1496	GRANDMOTHER
+Steinthal-1875-1565-344	344	Grossmutter	Großmutter väterlicherseits	paternal grandmother	3015	PATERNAL GRANDMOTHER
+Steinthal-1875-1565-345	345	Grossmutter	Großmutter mütterlicherseits	maternal grandmother	2177	MATERNAL GRANDMOTHER
```

Comment:
'Grossmutter' is not specified anyhow in the source. I know it would be logical if it was but I would stick to the source and leave just general GRANDMOTHER entry.

Reply:
How dare you poke at the exact things I was brooding about while mapping ;-)
The reason I still specified Grossmutter is because many concepts are not spelled out in full in this list (see also the Schwieger- terms in 356 onwards). But if Steinthal gives the hint to non-linguists to check whether the same term applies for maternal and paternal grandfathers, it would make sense to do the same with grandmothers (and the same for the following female kinship terms). He abbreviated so much that it would not make sense to spell out (väterlicherseits, mütterlicherseits) every time. But I agree, officially, the terms are not in there for the female family members. What do the others say, @chrzyki and @Schweikhard?